templates/job-runner/Dockerfile

```
@@ -0,0 +1,24 @@
+FROM liferay/base:latest
+
+ARG LABEL_BUILD_DATE
+ARG LABEL_NAME
+ARG LABEL_VCS_REF
+ARG LABEL_VCS_URL
+ARG LABEL_VERSION
+
+RUN apt-get install cron tini && \
+	apt-get clean
+
+COPY scripts/* /usr/local/bin/
+
+ENTRYPOINT ["/tini", "-v", "--", "/usr/local/bin/liferay_job_runner_entrypoint.sh"]
```

```suggestion
ENTRYPOINT ["tini", "-v", "--", "/usr/local/bin/liferay_job_runner_entrypoint.sh"]
```
`tini` isn't in the root anymore so we need to remove the `/`